src/pybind/mgr/cephadm/module.py

```diff
@@ -392,6 +413,21 @@ def _kick_serve_loop(self):
         self.log.debug('_kick_serve_loop')
         self.event.set()
 
+    # function responsible for logging single host into custom registry
+    def _registry_login(self, host, url, username, password):
+        self.log.debug(f"Attempting to log host {host} into custom registry @ {url}")
+        out, err, code = self._run_cephadm(
+            host, 'mon', 'registry-login',
+            [
+                '--registry-url', url,
+                '--registry-username', username,
+                '--registry-password', password,
```

Comment:
this needs be passed via stdin.

Reply:
Is there a standard procedure for taking arguments from stdin inside the module? I've been having a bit of trouble making that work.